src/elastio-snap.c

```diff
@@ -4913,6 +4930,7 @@ static int ioctl_elastio_snap_info(struct elastio_snap_info *info){
 	if(ret) goto error;
 
 	dev = snap_devices[info->minor];
+	wait_for_bio_complete(dev);
```

Comment:
`ioctl_elastio_snap_info` is called by `elastio` CLI many times in a row during snapshot. As I understood, this change allows to get response with an error, if it just appeared. And a small delay in response doesn't harm. And it will not affect performance of the snapshot read operations.
@skypodolsky please correct me, if I'm wrong.

Reply:
This is correct. Nevertheless, I decided to remove `wait_for_bio_complete()` to improve the response time if the system is overloaded with bio requests.